spec/spec_helper_acceptance.rb

```diff
@@ -55,6 +59,9 @@ def solaris_agents
 
       # Until solaris gets new image we need to add to the cert chain on solaris, call a beaker-puppet setup script to handle this
       hosts.each do |host|
+        if hosts.platform.match? %r{ubuntu}
```

Comment:
Rubocop is flagging this, could you update this to call `include?` instead of `match?`. Thanks!